src/app/Reducers/userReducer.js

```diff
@@ -0,0 +1,22 @@
+import { ADD_FRIEND, REMOVE_FRIEND } from '../Constants/ActionTypes';
+
+const initialState = {
+  friends: [{id: 1989}]
+};
+
+export default function userReducer(state=initialState, action) {
```

Comment:
create an initial state
always pass an state and action to the reducer (the reducer is not a store since it doesn't store the state)
it applies the action to the current state and returns the next state